src/pages/home/report/ReportActionCompose.js

```diff
@@ -446,6 +441,53 @@ class ReportActionCompose extends React.Component {
         ];
     }
 
+    /**
+     * Build the suggestions for mentions
+     * @param {Object} personalDetails
+     * @param {String} [searchValue]
+     * @returns {Object}
+     */
+    getMentionOptions(personalDetails, searchValue = '') {
+        const suggestions = [];
+
+        if (CONST.MENTION_SUGGESTER.HERE_TEXT.includes(searchValue)) {
```

Comment:
Should be `CONST.AUTO_COMPLETE_SUGGESTER.HERE_TEXT`, just realized it already exists

Reply:
Oh, I meant `CONST.AUTO_COMPLETE_SUGGESTER` already exists, let's move `HERE_TEXT` there